utils/azureCredentialsManager.ts

```diff
@@ -1,15 +1,15 @@
-import { SubscriptionClient, ResourceManagementClient, SubscriptionModels } from 'azure-arm-resource';
-import { AzureAccount } from '../typings/azure-account.api';
-import { ServiceClientCredentials } from 'ms-rest';
-import { AsyncPool } from '../utils/asyncpool';
 import { ContainerRegistryManagementClient } from 'azure-arm-containerregistry';
```

Comment:
I would untrack changes to this file as they are all tslint updates which will  come in once the azureCredentialsManager is merged into the Microsoft master branch